docs/system/k8s-metrics.md

```diff
@@ -329,5 +329,161 @@ This metric is [recommended][MetricRecommended].
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
+### Metric: `k8s.job.active_pods`
```

Comment:
nit and can totally be done as a follow-up. 
There are around ~20 metrics in this file and it might be hard to navigate around. It's also useful to see all metrics in the same place, so I wonder if it's possible to add a TOC here

Ptal at [messaging](https://github.com/open-telemetry/semantic-conventions/blob/b1691ad003f25a7b6ee58cee3de4ef4805bff8f6/docs/messaging/messaging-metrics.md#semantic-conventions-for-messaging-client-metrics) or [db](https://github.com/open-telemetry/semantic-conventions/blob/b1691ad003f25a7b6ee58cee3de4ef4805bff8f6/docs/database/database-metrics.md) to see how convenient it could be.

Reply:
Yeap, it was already in place from previous PRs. After rebasing it's included here as well.